### PR TITLE
add docs for batch endpoint

### PR DESCRIPTION
**Pull request by joephylum**
Adds docs for new `/batch` threat feed end point. Also adds docs about top level key additions to single package end point.

Note that I will update the example in the docs with one that actually returns slightly more interesting results after this gets deployed and I can run it against prod instead of staging.

### CODE REVIEW

File: docs/knowledge_base/threat_feed.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ The Phylum threat feed provides a curated view into malware being released acros
...
+   PHYLUM_API=p0_...
+   ```
 
 2. Use your API key to retrieve the latest packages in the threat feed:
 
-    ```bash
-    curl https://threats.phylum.io -H "Authorization: Bearer $PHYLUM_API"
-    ```
+   ```bash
+   curl https://threats.phylum.io -H "Authorization: Bearer $PHYLUM_API"
```
Comment (janasheehan) [inline docs/knowledge_base/threat_feed.md:18]: who does have access to the thread feed?

**Review by janasheehan — COMMENTED** [review]
perhaps i am the wrong person to review this...

File: docs/knowledge_base/threat_feed.md (hunk 1 above)
Comment (janasheehan) [reply docs/knowledge_base/threat_feed.md:18]:
```{"code":401,"description":"You do not have access to the threat feed!","name":"error"}

File: docs/knowledge_base/threat_feed.md [hunk 2]
```diff
@@ -157,13 +161,124 @@ Response:
     "Triaged Malware Rule": true
   },
   "malware": true,
+  "name": "adrmdr",
```
[the remark this answers is not in the document]
Comment (joephylum) [reply docs/knowledge_base/threat_feed.md:164]: This is just the name of a real package in pypi that I used in the example.

File: docs/knowledge_base/threat_feed.md [hunk 3]
```diff
@@ -157,13 +161,124 @@ Response:
...
+}
+```
+
+## Batch Package Lookup Endpoint
+
+The threat feed API also provides a batch package lookup endpoint. The response is the same as that from the single package endpoint but you'll receive a json payload for each package submitted in the bulk query.
+
+### Endpoint
+
+`POST https://threats.phylum.io/batch`
```
[the remark this answers is not in the document]
Comment (joephylum) [reply docs/knowledge_base/threat_feed.md:183]: It is not available in staging. You can run a local version of it, but no, it's not currently deployed in staging.

File: docs/knowledge_base/threat_feed.md [hunk 4]
```diff
@@ -157,13 +161,124 @@ Response:
...
+### Usage Example
+
+Request:
+
+```bash
+curl -X POST -H "Content-Type: application/json" -d '[
+["npm", "lodash", "4.17.21"],
+["pypi", "requests", "2.32.3"],
+["cargo", "serde", "1.0.203"]
+]' http://localhost:3000/batch -H "Authorization: Bearer $PHYLUM_API"
```
[the remark this answers is not in the document]
Comment (joephylum) [reply docs/knowledge_base/threat_feed.md:224]: No, good catch. It should be `https://threats.phylum.io/batch`. I'll fix!

File: docs/knowledge_base/threat_feed.md (hunk 1 above)
Comment (joephylum) [reply docs/knowledge_base/threat_feed.md:18]: @louislang would just have to add you to the authorized list.

Comment (janasheehan) [reply docs/knowledge_base/threat_feed.md:18]: aha! i shall request :-)

File: docs/knowledge_base/threat_feed.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (janasheehan) [reply docs/knowledge_base/threat_feed.md:164]: oh... duh. told you i am not the best to review this 😝

**Comment by joephylum** [comment]
Thanks @janasheehan! Your name was randomly picked to review when I opened the PR, so please don't feel obligated to review it too closely! I'll request some others with more specific knowledge of the endpoint. Thanks again!

**Review by ejortega — APPROVED** [review]
LGTM

File: docs/knowledge_base/threat_feed.md [hunk 5]
```diff
@@ -157,13 +161,124 @@ Response:
...
   "riskScores": {
     "author": 1.0,
     "engineering": 1.0,
     "license": 1.0,
     "malicious_code": 0.01,
     "total": 0.05,
     "vulnerability": 1.0
-  }
+  },
+  "version": "0.1.0"
```
Comment (ejortega) [inline docs/knowledge_base/threat_feed.md:173]: Maybe a nitpick, but maybe the response can match same order specified?

- `name`: The name of the package.
 - `version`: The version of the package.
 - `ecosystem`: The ecosystem to which the package was published.
 - `code`: The HTTP status code.
 - `malware`: A boolean indicating if the package has been triaged as malware.
 - `riskScores`: An object containing risk scores for each of the 5 Phylum risk domains associated with the pa

**Comment by furi0us333** [comment]
Please hold off on merging/publishing this for now.

**Comment by joephylum** [comment]
> Please hold off on merging/publishing this for now.

Sounds good! Will convert to draft just to make sure it doesn't get merged accidentally.

**Comment by furi0us333** [comment]
I am going to close this PR as we will not be including this text in the published docs.

**Pull request closed without merging**